frontend/src/views/auth/Signup.vue

```diff
@@ -222,6 +264,10 @@ export default defineComponent({
       name: "",
       nameManuallyEdited: false,
       acceptTermsAndPolicy: true,
+      showPassword: {
```

Comment:
Just use a single showPassword state to control both

Reply:
PTAL